mail_tracking/models/mail_thread.py

```diff
@@ -33,7 +33,8 @@ def message_get_suggested_recipients(self):
                     partner_id = record.message_partner_info_from_emails(
                         [email_parts[1]])[0].get('partner_id')
                     if not partner_id:
-                        res[record.id].append((False, cc, _('Cc')))
+                        record._message_add_suggested_recipient(
```

Comment:
How performant is this? Isn't better to have a set?

Reply:
Done, thanks!